### PR TITLE
generate: add new cgroup namespace option

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: man/ocitools-generate.1.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ compatible runtime like runC to run a container.
 **--cap-drop**=[]
   Drop Linux capabilities
 
+**--cgroup**=[*PATH*]
+  Use an Cgroup namespace.  If *PATH* is set, join that namespace.  If it
```
Comment (mrunalp) [inline man/ocitools-generate.1.md]: s/an/a

**Comment by wking** [comment]
With the fix in [1], 71c8a6b will look good to me.  Good catch on the
Bash-completion fixes.

[1]: https://github.com/opencontainers/ocitools/pull/122#discussion_r68432337

File: man/ocitools-generate.1.md (hunk 1 above)
Comment (Mashimiao) [inline man/ocitools-generate.1.md]: Thanks, fixed.

**Comment by Mashimiao** [comment]
ping @liangchenye  @mrunalp

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**